### DISCUSSION TITLE
more than 10 ws icons

### DISCUSSION THREAD

**Discussion by kewur** [polybar/polybar#2661] [Q&A]
Hi, 

I'm wondering if we can support more than 10 workspaces. I have 3 monitors with the setup 3 x 4 x 3 workspaces distributed among them but I find that it is not enough for me. anybody has the same issue?

**Comment by patrick96** [comment]
This is something you have to configure in your window manager. polybar has no control over how many workspaces exist and it shouldn't have a problem displaying even 100 workspaces
